solutions/01_hello_html/hello.js

```diff
@@ -0,0 +1 @@
+console.log("Hello, World!")
```

Comment:
Semicolons aren't required in JS, but I strongly encourage you to make it a habit and always use 'em